deploy/doc/config.md

```diff
@@ -0,0 +1,69 @@
+[//]: # (This file is built out of deploy/doc/config.md, please do not edit it manually)  
+[//]: # (To rebuild it run `make config`)
+
+# Configuration
+
+## ClickHouse `[clickhouse]`
+
+### URL `url`
+Detailed explanation of ClickHouse HTTP interface is given in [documentation](https://clickhouse.tech/docs/en/interfaces/http). It's recommended to create a dedicated read-only used for graphite-clickhouse.
+
+Example: `url = "http://graphite:qwerty@localhost:8123/?readonly=2&log_queries=1"`
+
+Some useful parameters:
+
+- [log_queries=1](https://clickhouse.tech/docs/en/operations/settings/settings/#settings-log-queries): all queries will be logged in the `system.query_log` table. Useful for debug.
+- [readonly=2](https://clickhouse.tech/docs/en/operations/settings/permissions-for-queries/#settings_readonly): do not change data on the server
+- [max_rows_to_read=200000000](https://clickhouse.tech/docs/en/operations/settings/query-complexity/#max-rows-to-read): useful if you want to prevent too broad requests
+- [cancel_http_readonly_queries_on_client_close=1](https://clickhouse.tech/docs/en/operations/settings/settings/#cancel-http-readonly-queries-on-client-close): cancel DB query when request is canceled.
+
+All these and more settings can be set in clickhouse-server configuration as user's profile settings.
+
+Useless settings:
+
+- `max_query_size`: after external data is used, the query are very short
+- `max_ast_elements`: the same
+- `max_execution_time`: with `cancel_http_readonly_queries_on_client_close=1` and `data-timeout = "1m"` it's already covered.
+
+### Index table
+See [index table](./index-table.md) documentation for details.
+
+### Index reversed queries tuning
+TODO: update after #158 or #160 is merged
+
+### ClickHouse aggregation
+For detailed description of `max-data-points` and `internal-aggregation` see [aggregation documentation](./aggregation.md).
+
+## Data tables `[[data-table]]`
+
+### Rollup
+The daemon used to use the XML-files with the same [graphite_rollup](https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/graphitemergetree/#rollup-configuration) scheme as used on ClickHouse server.
```

Comment:
"used to use" - overcomplicated, I would maybe start this section with what is recommended, then mention "Previous best option is to use same format XML configuration as clickhouse use" or something like that.

"XML-files with the same" - maybe `same XML format as`? Also AFAIR it's possible to just point to the same file, if it's extracted from the main config.

Reply:
Also maybe it's good idea to tell a few words why it's needed? (that aggregation on latest data happens on graphite-clickhouse side)

Reply:
I guess, it's better now